discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -210,13 +229,18 @@ def download_infraenv_image(self, infraenv_id: str, image_path: str) -> None:
         response_obj = response[0]
         self._download(response=response_obj, file_path=image_path, verify_file_size=True)
 
+#TODO remove and use infraenv
     def update_hosts(
             self, cluster_id: str, hosts_with_roles, hosts_names: Optional[models.ClusterupdateparamsHostsNames] = None
     ) -> models.cluster.Cluster:
```

Comment:
Add deprecation message instead 

```suggestion
#TODO remove and use infraenv
    def update_hosts(
            self, cluster_id: str, hosts_with_roles, hosts_names: Optional[models.ClusterupdateparamsHostsNames] = None
    ) -> models.cluster.Cluster:
    warnings.warn("update_hosts is deprecated. Use update_host instead.", DeprecationWarning)
```

Reply:
added